README.md

```diff
@@ -1,407 +1,436 @@
-nnoremap <silent> gr <cmd>lua require'telescope.builtin'.lsp_references{ shorten_path = true }<CR>
-```
+| Description                                           | Keys                   | Options                    |
+|-------------------------------------------------------|------------------------|----------------------------|
+| Where the prompt should be located.                   | `prompt_position`      | top/bottom                 |
+| What should the prompt prefix be.                     | `prompt_prefix`        | string                     |
+| Where first selection should be located.              | `sorting_strategy`     | descending/ascending       |
+| How the telescope is drawn.                           | `layout_strategy`      | center/horizontal/vertical |
+| How transparent is the telescope window should be.    | `winblend`             | NUM                        |
+| Layout specific configuration ........ TODO           | `layout_defaults`      | TODO                       |
+| TODO                                                  | `width`                | NUM                        |
+| TODO                                                  | `preview_cutoff`       | NUM                        |
+| TODO                                                  | `results_height`       | NUM                        |
+| TODO                                                  | `results_width`        | NUM                        |
+| The border chars, it gives border telescope window    | `borderchars`          | dict                       | 
+| Whether to color devicons or not                      | `color_devicons`       | boolean                    |
+| Whether to use less or bat .. TODO                    | `use_less`             | boolean                    |
 
-What this does:
+#### Sorting
 
-* Make the paths full size by default. On LSP references we are shortening paths.
-* Bind `<ctrl-p>` for a common mapping to find files.
-  - Using `telescope.builtin.git_files` is better in git directories. You can make a toggle to detect if it's a git directory.
-* Bind `gr` to find references in LSP.
-  - `telescope.builtin.lsp_workspace_symbols` and `telescope.builtin.lsp_document_symbols` are also good to bind for LSP.
+| Description                                           | Keys                   | Options                    |
+|-------------------------------------------------------|------------------------|----------------------------|
+| The sorter for file lists.                            | `file_sorter`          | [see sorters](#built-in-sorters)    |
+| The sorter for everything else.                       | `generic_sorter`       | [see sorters](#built-in-sorters)    |
+| The command line argument for grep search ... TODO.   | `vimgrep_arguments`    | dict                               |
+| ... TODO                                              | `selection_strategy`   | follow/reset/row                   | 
```

Comment:
What happens to the selection if the list changes. For example i'm at index 5 and than in start typing, which results into a result change, `reset` will put the selection back to the beginning, `row` will keep the selection at this index (or the last possible index) and `follow` will result in following the selected element, so the selection might jump when the element jumps. Is this clear? I don't know how to explain this easily for the readme

Reply:
It makes sense :+1: , I've added to the column: What happens to the selection if the list changes.

I guess it could be improved, but for now its good enough I guess.